new-website/deepchem/components/Header/NavLinks.js

```diff
@@ -48,6 +48,13 @@ const NavLinks = () => {
             blank={true}
           />
         </li>
+        <li>
+          <NavLink
```

Comment:
@Cannon07 Please add a screenshot of what this looks like

Reply:
![screenshot](https://github.com/deepchem/deepchem.github.io/assets/73459690/180402ab-82de-4ba6-b361-3e71e1a26f00)